File: frontend/src/features/actividades/registrar_tasacion/models/tasacion_detalle.ts
import type { TipoTasacion } from "./registrar_tasacion";

export interface TasacionDetalle {
  id_tasacion_detalle: number;
  id_tasacion: number;
  id_expediente: number;
  tipo_tasacion: boolean;
  nro_tasacion_p1: string | null;
  nro_tasacion_p2: string | null;
  nro_tasacion_p3: string | null;
  superficie_edificada: string | null;
  superficie_terreno: string | null;
  fecha_informe_tasacion: string | null;
  fecha_solicitud_tasacion: string | null;
  fecha_recepcion_tasacion: string | null;
  valor_tasacion_uf: number | null;
  valor_tasacion_pesos: number | null;
  valor_liquidacion_uf: number | null;
  valor_liquidacion_pesos: number | null;
  monto_asegurable_uf: number | null;
  monto_asegurable_pesos: number | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by: number | null;
  modified_date: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/tasacion_detalle.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class tasacion_detalle : base_auditoria
    {
        public int id_tasacion_detalle { get; set; }
        public int id_tasacion { get; set; }
        public long id_expediente { get; set; }
        public string? tipo_tasacion { get; set; }
        public string? nro_tasacion_p1 { get; set; }
        public string? nro_tasacion_p2 { get; set; }
        public string? nro_tasacion_p3 { get; set; }
        public string? superficie_edificada { get; set; }
        public string? superficie_terreno { get; set; }
        public DateTime? fecha_informe_tasacion { get; set; }
        public DateTime? fecha_solicitud_tasacion { get; set; }
        public DateTime? fecha_recepcion_tasacion { get; set; }
        public decimal? valor_tasacion_uf { get; set; }
        public decimal? valor_tasacion_pesos { get; set; }
        public decimal? valor_liquidacion_uf { get; set; }
        public decimal? valor_liquidacion_pesos { get; set; }
        public decimal? monto_asegurable_uf { get; set; }
        public decimal? monto_asegurable_pesos { get; set; }
    }
}
